lib/components/app/app-menu.js

```diff
@@ -52,47 +142,69 @@ class AppMenu extends Component {
       toggleMailables
     } = this.props
 
+    const { isPaneOpen } = this.state
     return (
-      <div aria-label='app-menu' className='app-menu' role='navigation'>
-        <DropdownButton
-          aria-label='Application Menu'
-          className='app-menu-button'
-          id='app-menu'
-          noCaret
-          title={(<Icon type='bars' />)}>
-          <MenuItem onClick={this._showRouteViewer}>
-            <Icon type='bus' /> {languageConfig.routeViewer || 'Route Viewer'}
-          </MenuItem>
-          {callTakerEnabled &&
-            <MenuItem onClick={resetAndToggleCallHistory}>
-              <Icon type='history' /> Call History
-            </MenuItem>
-          }
-          {fieldTripEnabled &&
-            <MenuItem onClick={resetAndToggleFieldTrips}>
-              <Icon type='graduation-cap' /> Field Trip
+      <>
+        <div
+          aria-label={isPaneOpen ? 'Close Menu' : 'Open Menu'}
+          className='app-menu-icon'
+          onClick={this._expandPane}
+          onKeyDown={this._expandPane}
+          role='button'
+          tabIndex={0}>
+          <span className={isPaneOpen ? 'menu-left-x' : 'menu-top-line'} />
+          <span className={isPaneOpen ? '' : 'menu-middle-line'} />
+          <span className={isPaneOpen ? 'menu-right-x' : 'menu-bottom-line'} />
+        </div>
+        <SlidingPane
+          from='left'
+          isOpen={isPaneOpen}
+          onRequestClose={this._expandPane}
+          width='320px'>
+          <ul className='app-menu'>
+            <MenuItem
+              className='menu-item'
+              onClick={this._showRouteViewer}>
+              <Icon name='bus' />{languageConfig.routeViewer || 'Route Viewer'}
```

Comment:
Think about changes needed to merge this with #435 once approved.

Reply:
Intl prep added in 542d756098ceae9ad56f74334fae242b4be7c938